pkg/storage/storage.go

```diff
@@ -37,10 +40,14 @@ func newDriver(crname string, crnamespace string, cfg *opapi.ImageRegistryConfig
 		drivers = append(drivers, azure.NewDriver(crname, crnamespace, cfg.Azure))
 	}
 
-	if cfg.Filesystem != nil {
+	if cfg.Filesystem != nil && cfg.Filesystem.VolumeSource.EmptyDir == nil {
```

Comment:
What's the difference between the `filesystem` and `emptydir` drivers?

Reply:
filesystem driver will (ultimately) create a PVC.  Perhaps it should be renamed to the persistentvolume driver.

emptydir sets up an emptydir volumesource.